internal/containerupdate/exec_updater.go

```diff
@@ -64,3 +69,15 @@ func (cu *ExecUpdater) UpdateContainer(ctx context.Context, cInfo store.Containe
 
 	return nil
 }
+
+func handleK8sExecError(out *bytes.Buffer, err error) error {
+	msg := strings.ToLower(fmt.Sprintf("%s\n%s", out.String(), err.Error()))
+	if strings.Contains(msg, "permission denied") || strings.Contains(msg, "cannot open") {
+		return fmt.Errorf("denied access to the container filesystem. Please check:\n"+
```

Comment:
Not a blocker and this is a big improvement over status quo, so feel free to ignore.

I found the sample output a bit confusing:
1) The error doesn't show what was interpreted as "denied access" - I found it confusing to read "denied access to the container filesystem" and then the evidence presented is "original error: command terminated with exit code 2"
2) I really expected the text following "with unexpected error:" to be the error. Can we move the error to the start of this or change the colon to a period?

Reply:
ya, i think this is more of a complaint about the way we assemble live-update errors - it's trying to combine errors from a lot of different sources and make sense of them, while also trying not to hide anything from the user. not sure what the right answer is here.

Reply:
here's what it looks like now

![Screenshot from 2021-02-17 12-04-11](https://user-images.githubusercontent.com/278641/108239836-4a9eb180-7118-11eb-9ce8-8621ab12e032.png)